app/classifier/classifier.jsx

```diff
@@ -175,7 +157,7 @@ class Classifier extends React.Component {
   }
 
   updateAnnotations() {
-    const { annotations } = this.props.classification;
+    const annotations = this.props.classification.annotations.slice();
     this.setState({ annotations });
     if (this.props.feedback.active) {
```

Comment:
not directly related to the changes in this PR, but in this demo project with feedback - https://remove-feedback-hack.pfe-preview.zooniverse.org/projects/markb-panoptes/feedback-testing-mb - if you first make a classification in workflow "Feedback 4" then go back to the project landing and try to go into workflow "Feedback 3" it hangs and doesn't load a subject, related to #4243.

I think changing this line to:
`if (annotations.length > 0 && this.props.feedback.active) {`
solves the issue.

Reply:
I can have a look into this bug tomorrow. If I understand correctly, it's a bug on master rather than a bug in this PR?

Reply:
that's correct, on master, not this PR, figure I'd mention if feel like including, but no worries if not

Reply:
Probably quicker to fix and merge that one separately, rather than fix it here. I'll have a look at it tomorrow but it sounds like a quick fix.

Reply:
Turns out it was easier to fix here. Should be fixed by 4bba343.

Reply:
This doesn't fix the bug in the feedback action itself, so I've opened #4275.